pkg/rpcclient/actor/doc_test.go

```diff
@@ -0,0 +1,122 @@
+	// have the script and a proper system fee for it, therefore SendUncheckedRun
+	// can be used.
+	txid, vub, _ := a.SendUncheckedRun(res.Script, res.GasConsumed, nil, nil)
+	_ = txid
+	_ = vub
+	// You need to wait for it to persist and then check the on-chain result of it.
+
+	// Now we want to send some transaction, but give it a priority by increasing
+	// its network fee, this can be done with Tuned APIs.
+	txid, vub, _ = a.SendTunedCall(customContract, "method", nil, func(r *result.Invoke, t *transaction.Transaction) error {
+		// This code is run after the test-invocation done by *Call methods.
+		// Reuse the default function to check for HALT execution state.
+		err := actor.DefaultCheckerModifier(r, t)
+		if err != nil {
+			return err
+		}
+		// Some additional checks can be performed right here, but we only
+		// want to raise the network fee by ~20%.
+		t.NetworkFee += (t.NetworkFee / 5)
+		return nil
+	}, 1, 2, 3)
+	_ = txid
+	_ = vub
+
+	// Actor can be used for higher-level wrappers as well, if we want to interact with
+	// NEO then [neo] package can accept our Actor and allow to easily use NEO methods.
+	neoContract := neo.New(a)
+	balance, _ := neoContract.BalanceOf(a.Sender())
+	_ = balance
+
+	// Now suppose the second wallet account is a committee account. We want to
+	// create and sign transactions for committee, but use the first account as
+	// a sender (because committee account has no GAS). We at the same time want
+	// to make all transactions using this actor high-priority ones, because
+	// committee can use this attribute.
+
+	// Get the default options to have CheckerModifier/Modifier set up correctly.
+	opts := actor.NewDefaultOptions()
+	// And override attributes.
+	opts.Attributes = []transaction.Attribute{{Type: transaction.HighPriority}}
+
+	// Create an Actor.
+	a, _ = actor.NewTuned(c, []actor.SignerAccount{{
+		// Sender, regular account with None scope.
+		Signer: transaction.Signer{
+			Account: w.Accounts[0].ScriptHash(),
+			Scopes:  transaction.None,
+		},
+		Account: w.Accounts[0],
+	}, {
+		// Commmitee.
+		Signer: transaction.Signer{
+			Account: w.Accounts[1].ScriptHash(),
+			Scopes:  transaction.CalledByEntry,
+		},
+		Account: w.Accounts[1],
+	}}, opts)
+
+	// Use policy contract wrapper to simplify things. All changes in the
+	// Policy contract are made by the committee.
+	policyContract := policy.New(a)
+
+	// Create a transaction to set storage price, it'll be high-priority and have two
+	// signers from above. Committee is a multisignature account, so we can't sign/send
+	// it right away, w.Accounts[1] has only one public key. Therefore, we need to
+	// create a partially signed transaction and save it, then collect other signatures
+	// and send.
+	tx, _ := policyContract.SetStoragePriceUnsigned(10)
+
+	net := a.GetNetwork()
+	scCtx := sccontext.NewParameterContext("Neo.Network.P2P.Payloads.Transaction", net, tx)
```

Comment:
We need to move this string to a separate constant, we use it from our code as far.

Reply:
Sure, I had this thought, but then was busy with something else. Fixed.